exchanges/gateio/gateio_wrapper.go

```diff
@@ -210,6 +210,8 @@ func (g *Gateio) Setup(exch *config.Exchange) error {
 		GenerateSubscriptions:  g.GenerateDefaultSubscriptions,
 		ConnectionMonitorDelay: exch.ConnectionMonitorDelay,
 		Features:               &g.Features.Supports.WebsocketCapabilities,
+		FillsFeed:              g.Features.Enabled.FillsFeed,
+		TradeFeed:              g.Features.Enabled.TradeFeed,
```

Comment:
This seems like fills is not set anywhere else in the code base? Suggestion: In stream.WebsocketSetup change Features from being a pointer to websocket capabilities, pass in the &g.Features and in the Setup call, apply it behind the scenes.